web/embed-html.html

```diff
@@ -33,19 +33,36 @@
           rel="stylesheet" media="screen">
     <link rel="import" href="embed_components.html">
     <script type="application/dart" src="embed.dart"></script>
+    <style>
+      #run-button {
+        background-color: #d23f31;
+      }
+
+      .button-bar {
```

Comment:
These styles are still relevant. We still maintain the div button bar. It just only contains 1 button now.